day-01/src/display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = (props) => {
```

Comment:
if u use ({ counter }) instead of (props), then u don't need the props later

Reply:
Best solution: 
```js
const Display = ({ counter }) => (
  <div> {counter} </div>
);
```
fix the Display and the Button components based on this solution :)